src/lib/fields/CreatibutorsField.js

```diff
@@ -12,36 +12,6 @@ import _set from 'lodash/set';
 import { Field } from './Field';
```

Comment:
This file is not used in RecordSerializer.js anymore, but only some of the code was deleted. Should it still be used? If not, we should delete this file.

Reply:
Removed!